### DISCUSSION TITLE
Nozzle too high after M600

### DISCUSSION THREAD

**Discussion by janwillem117** [Jyers/Marlin#888] [Q&A] [answered]
I've had a first try at printing with 2 colors with the M600 command on my Ender 3 V2 with Jyers 1.3.1 firmware. The problem is however that after changing the fillament the printer resumes printing but the nozzle is too far away from my print(Probably 1 or 2mm). I tried to quickly change the Z offset to lower the nozzle but the print was already ruined. I sliced my model using Prusaslicer. Any idea how to fix this?

**Comment by Cal-LF** [comment, marked as answer]
Next release should fix this.

**Reply by janwillem117** [reply]
I have just tried 1.3.3 but that did not change anything. Or did you mean a future release?

**Reply by janwillem117** [reply]
> 
> 
> Next release should fix this.

I have just tried 1.3.4 and the issue remains the same. Should I do anything else than just put the bin file on the sd and start the printer?

Here is a video of the issue:
[https://youtu.be/5WEROs4Bq8I](url)

**Reply by Jyers** [reply]
Hmmmm, that seems slight enough that it could be a hardware issue. For sanities sake try clearing eeprom by going to the control menu, selecting reset to defaults and then store configuration. If that doesn't work check the z position on the display before and after the park to see if it thinks its going back to the right spot.

**Reply by janwillem117** [reply]
I did as you suggested. I am printing with 0.28mm layer height. It stopped at 3.0mm and resumed at 3.28mm so that seems fine..

I should also mention that I am having this issue on both my Ender 3 V2’s.

**Reply by Jyers** [reply]
If the printer thinks it's going back to the right height, but it's too high, that is almost definitely a hardware issue. Perhaps your z axis is too tight causing slipping?

**Reply by janwillem117** [reply]
I will have a look but that seems a bit weird since I am having this issue on both printers. Or I did something wrong assembling them.. 😅

**Reply by Jyers** [reply]
It does seem strange, but I haven't had this report from anyone else so it seems to be an isolated issue of some kind

**Reply by janwillem117** [reply]
I have finally resolved the issue. One of the eccentric wheel nuts was too tight which led to the right z axis not moving correctly.

**Comment by smooreace** [comment]
Ive had zero trouble with it... now at least 30-35 times Ive swapped colors
mid-print. Using Cura though.... not that that should have anything to do
with it

**Comment by embraceentropy** [comment]
I was having the same issue. If you don't want to wait for the next release see here: https://github.com/Jyers/Marlin/discussions/714

Prior to this I was only using pre-compiled FW. This was the point I started messing around with self-compiling and now I'm obsessed!

**Comment by smooreace** [comment]
I didnt mention that Im running self-compiled firmware based on data
available on 11 May.
